appcues/src/main/java/com/appcues/debugger/screencapture/AndroidTargetingStrategy.kt

```diff
@@ -54,6 +68,10 @@ internal data class AndroidViewSelector(
                 weight += 100
             }
 
+            if (it.appcuesId != null && it.appcuesId == appcuesId) {
+                weight += 100
+            }
```

Comment:
thinking we should make this match the [iOS side](https://github.com/appcues/appcues-ios-sdk/blob/main/Sources/AppcuesKit/Presentation/Debugger/ScreenCapturing/UIKitElementTargeting.swift#L50-L52) where appcuesID matches the highest level selector (so the 1000 case here matching resource name). In practice, right now it would always be mutually exclusive, granted.